pkg/fleet-manager/application/rollout_helper.go

```diff
@@ -167,6 +167,48 @@ func (a *ApplicationManager) syncRolloutPolicyForCluster(ctx context.Context,
 	return ctrl.Result{}, nil
 }
 
+func (a *ApplicationManager) deleteResourcesInClusters(ctx context.Context, app *applicationapi.Application, fleet *fleetapi.Fleet) error {
+	log := ctrl.LoggerFrom(ctx)
+
+	for _, syncPolicy := range app.Spec.SyncPolicies {
```

Comment:
maybe it is handled in other place， but here i cannot see how we delete the resources when we update application by removing `SyncPolicies`